project/PACKAGERS.md

```diff
@@ -1,7 +1,19 @@
-# Dear Packager,
+# For Users
+
+The Moby Project is widely packaged by many independent software distributors,
+generally under the name 'docker' or 'docker.io.'
+
+A non-exhuastive list of known software packagers and distributions includes:
```

Comment:
How exhaustive do you want this list to be?  Repology has a [fairly long list](https://repology.org/project/docker/versions) (though [it](https://repology.org/project/docker.io-app/versions) [isn't](https://repology.org/project/docker-engine/versions) [fully](https://repology.org/project/dockerd/versions) [normalized](https://repology.org/project/docker-ce/versions)).  Beyond those lists, I also know of [Bottlerocket](https://github.com/bottlerocket-os/bottlerocket/tree/develop/packages/docker-engine) and [Google Container-Optimized OS](https://cos.googlesource.com/cos/overlays/board-overlays/+/refs/heads/master/project-lakitu/app-containers/docker/).

Reply:
This is definitely pretty initial; it definitely would be helpful to at least document say, the top 10 Linux distributions with packaging, and any non-distribution distributions that are well known.

Interesting to know that Bottlerocket has separate packaging from AL -- thanks for pointing that out! I'll try and add some more entries from the links you have provided.